flight/Modules/FlightStats/flightstatsmodule.c

```diff
@@ -51,14 +51,13 @@
 // Private variables
 static bool module_enabled;
 static struct pios_thread *flightStatsTaskHandle;
-static FlightStatsSettingsData settings;
+static volatile FlightStatsSettingsData settings;
```

Comment:
If you don't make it volatile you get a compile error (because the type is volatile in the registration function for the copy-callback registration).  This is at least correct, too, where the old version wasn't and could lead to undefined behavior.